Add additional comments and examples for deploy and undeploy commands (#39)
NearNodeFlash/nnf-deploy: Readme.md

## nnf-deploy

> nnf-deploy is a golang executable capable of building components of the Rabbit software stack locally as well as deploying and un-deploying those components to a k8s cluster specified by the current kube config.

Reviewer: ```suggestion
nnf-deploy is a golang executable capable of building components of the Rabbit software stack locally as well as deploying and un-deploying those components to a k8s cluster specified by the current kubeconfig.
```

## Deploy

Deploy all the submodules using the `deploy` command

```bash
./nnf-deploy deploy
```

> To only deploy specific repositories, include the desired modules after `deploy` command. For example, to only deploy `dws` and `nnf-sos` repositories, use

Reviewer: ```suggestion
To deploy only specific repositories, include the desired modules after `deploy` command. For example, to deploy only `dws` and `nnf-sos` repositories, use
```

```bash
./nnf-deploy deploy dws nnf-sos
```

> 

Author: arguments can be in any order. The tool itself deploys in the order of the `modules` array

## Undeploy

> **WARNING!** Undeploy should be preceded by deleting any user or administrator created resources such as `lustrefilesystems` and `workflows` using kubectl commands

Reviewer: ```suggestion
**WARNING!** Before you undeploy, delete any user or administrator created resources such as `lustrefilesystems` and `workflows` using kubectl commands
```

```bash
kubectl delete workflows.dws.cray.hpe.com --all
kubectl delete lustrefilesystems.cray.hpe.com --all
```

Undeploy all the submodules using the `undeploy` command.

```bash
./nnf-deploy undeploy
```

> Similar to deploy, undeploy specific repositories by including the desired modules after the `undeploy` command. For example, to only undeploy `dws` and `nnf-sos`, use

Reviewer: ```suggestion
Similar to deploy, you may undeploy specific repositories by including the desired modules after the `undeploy` command. For example, to undeploy only `dws` and `nnf-sos`, use
```
